src/beer.py

```diff
@@ -17,3 +18,14 @@ def get_style(self):
 
     def get_price(self):
         return self.price
+
+    def get_quantity(self):
+        return self.get_quantity()
+
+    def is_available(self):
```

Comment:
This needs a test.  While you are at it, add a simple test for the other fields - just for coverage.